drivers/crypto/ksecdd/stubs.c

```diff
@@ -160,11 +161,15 @@ GenerateSessionKey(VOID)
     UNIMPLEMENTED_DBGBREAK();
 }
 
-VOID
+SECURITY_STATUS
 SEC_ENTRY
-GetSecurityUserInfo(VOID)
+GetSecurityUserInfo(
+    _In_opt_ PLUID LogonId,
+    _In_     ULONG Flags,
+    _Out_    PSecurityUserData *UserInformation)
 {
-    UNIMPLEMENTED_DBGBREAK();
+    UNIMPLEMENTED;
+    return STATUS_UNSUCCESSFUL;
```

Comment:
```suggestion
    *UserInformation = NULL;
    return STATUS_UNSUCCESSFUL;
```

Reply:
UserInformation is not listed as optional, so a null check is not required.

Reply:
@learn-more, so is it still a good idea to NULL out the *UsesrInformation since I will not be returning anything here?